src/qt/sendcoinsdialog.h

```diff
@@ -100,13 +100,14 @@ private Q_SLOTS:
 };
 
 
+#define SEND_CONFIRM_DELAY   3
 
 class SendConfirmationDialog : public QMessageBox
 {
     Q_OBJECT
 
 public:
-    SendConfirmationDialog(const QString &title, const QString &text, int secDelay = 0, QWidget *parent = 0);
+    SendConfirmationDialog(const QString &title, const QString &text, int secDelay = SEND_CONFIRM_DELAY, QWidget *parent = 0);
```

Comment:
> Or do I misunderstand something?

No you're right. I didn't see sendcoinsdialog is also passing a this argument, so there is no way to use the default delay argument.